src/main/kotlin/pl/put/smartgarden/domain/user/dto/UserDto.kt

```diff
@@ -0,0 +1,51 @@
+package pl.put.smartgarden.domain.user.dto
+
+import javax.validation.constraints.Email
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.Pattern
+import javax.validation.constraints.Size
+
+data class UserSignUpDto(
+        @field:Size(min = 3, max = 25, message = "{validation.user.username.min}")
+        @field:Pattern(regexp = "^[a-zA-Z]+$", message = "{validation.user.username.letters}")
+        val username: String,
+        @field:Email(message = "{validation.user.email.not-valid}")
+        @field:NotBlank(message = "{validation.user.email.not-empty}")
+        val email: String,
+        @field:NotBlank(message = "{validation.user.password.not-empty}")
+        val password: String,
+        val deviceGuid: String
```

Comment:
przy rejestracji już chcemy podać guid?

Reply:
Jeśli dobrze pamiętam to chciałem aby to pole było opcjonalne przy rejestracji (bo w tym wspólnym docu było chyba wpisane)